packages/react-wildcat/src/middleware/renderReactWithWebpack.js

```diff
@@ -52,6 +61,7 @@ module.exports = function renderReactWithWebpack(root, options) {
 
             if (displayBlueBoxOfDeath) {
```

Comment:
Perhaps instead of mutating state you can use the blue box of death to display the overlay.